R/workhorse.R

```diff
@@ -34,9 +31,8 @@ set_path <- function(x) {
   if (!is.null(x$path)) {
     return(x)
   }
-  ## find the path that matches the given method and resource
-  x$path <- unique(.drive$params$path[.drive$params$method == x$method &
-                                        .drive$params$resource == x$resource])
+  ## find the path that matches the given endpoint
+  x$path <- unique(.drive$params$path[.drive$params$endpoint == x$endpoint])
   return(x)
```

Comment:
right now `.drive.params` is a tibble that has a separate row with every parameter allowed for each endpoint, so it is just grabbing the endpoints is >1 (but they should all have the same value). I could save it in 2 separate tibbles (one for the paths, which are unique to each endpoint and one for the parameters?)